cmd/zt_remove_adls_test.go

```diff
@@ -244,3 +244,178 @@ func (s *cmdIntegrationSuite) TestRemoveListOfALDSFilesWithIncludeExclude(c *chk
 		c.Assert(strings.Contains(err.Error(), "exclude"), chk.Equals, true)
 	})
 }
+
+func (s *cmdIntegrationSuite) TestRemoveFilesystemWithFromTo(c *chk.C) {
+	// invoke the interceptor so lifecycle manager does not shut down the tests
+	mockedRPC := interceptor{}
+	mockedRPC.init()
+	ctx := context.Background()
+
+	// get service SAS for raw input
+	serviceURLWithSAS := scenarioHelper{}.getRawAdlsServiceURLWithSAS(c)
+
+	// set up the filesystem to be deleted
+	bfsServiceURL := GetBFSSU()
```

Comment:
The code between line 258-267 is redundant in all the ADLS Gen2 UTs you've written. Let's wrap these lines in one common function and call that function instead of writing the same lines again and again.